Home_Page.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
+    <link rel="stylesheet" href="Style_of_Pages.css">
+    <script src="scripts_Home.js"></script>
+    <style>
```

Comment:
while this definitely works, consider avoiding in-line style tags in an HTML page. would this work in the stylesheet that you've linked?